test/unittests/mongoBackend/mongoContextProvidersQuery_test.cpp

```diff
@@ -1586,8 +2192,8 @@ TEST(mongoContextProvidersQueryRequest, severalCprs2)
 * Result: E1 - A1 - 1
 *              A2 - fwd CPR1
 *              A3 - fwd CPR2
-*              <null> - fwd CPR2
-*              <null> - fwd CPR3
+*
+*         CPR vector: [CPR2, CPR3]
```

Comment:
As disccussed time ago, "general" entity-level CPr is now a list instead of a single value.